charts/common/kubeflow-istio-resources/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "1.14.1"
```

Comment:
I think these are specific to Kubeflow so v1.6.1 is more appropriate, same for cluster-local-gateway

Reply:
ok